File: app/dbmodels/Notepad.php
<?php

$table = array(
    'fields'        => array(),
    'oldfields'     => array(),
    'renamefields'  => array(),
    'indexes'       => array(),
    'oldindexes'    => array(),
    'relationships' => array(),
    'model'         => 'Notepad',
    'menu'          => ''
);
$table['fields'] = array(
    'id'                    => DBModels::type('increments', 
                                              array('visible' => false)),
    'item_type'              => DBModels::type('string', 
                                              array('visible' => false,
                                                    'size'    => 100)),
    'item_id'               => DBModels::type('integer', 
                                             array('visible' => false)),
    'user_id'               => DBModels::type('integer'),
    'user_lastupdater_id'   => DBModels::type('integer'),
    'content'               => DBModels::type('longtext'),
    
                                         
);
  
  
$table['relationships'] = array(
    'user_id' => array(
        'type' => 'belongsTo',
        'item' => 'User'),
    'user_lastupdater_id' => array(
        'type' => 'belongsTo',
        'item' => 'User'),
);

File: fusionglpi/database/notepad.py
from fusionglpi import db


class Notepad(db.Model):
    id = db.Column(db.Integer, primary_key=True)
    item_type = db.Column(db.String(100, None, True))
    item_id = db.Column(db.Integer, nullable=False, server_default="0")
    user_id = db.Column(db.Integer, db.ForeignKey('user.id'))
    user_lastupdater_id = db.Column(db.Integer, nullable=False,
                                    server_default="0")
    content = db.Column(db.Text)
